components/devtools/actors/inspector/style_rule.rs

```diff
@@ -185,18 +185,16 @@ impl StyleRuleActor {
             css_text: "".into(), // TODO: Specify the css text
             declarations: style
                 .into_iter()
-                .filter_map(|decl| {
-                    Some(AppliedDeclaration {
-                        colon_offsets: vec![],
-                        is_name_valid: true,
-                        is_used: IsUsed { used: true },
-                        is_valid: true,
-                        name: decl.name,
-                        offsets: vec![], // TODO: Get the source of the declaration
```

Comment:
Could you please add the "TODO" comment back? It usually means that there is some unfinished business! But otherwise looks good c:

Also, you need to sign your commits, if you go to "Checks" or [here](https://github.com/servo/servo/pull/33693/checks?check_run_id=31188045885) it has instructions on the git command to run.

Reply:
Can you please check if its okay now...